datalad/distribution/remove.py

```diff
@@ -94,6 +98,8 @@ def __call__(
             dataset=None,
             recursive=False,
             check=True,
+            save=True,
+            message=None,
```

Comment:
I think it's inconsistent to have `save`/`nosave` (and consequently `message`) in `remove` but not in `add`.
Those are not meant to be commands that leave some staged things behind. ATM, I feel this undermines the API concept.

Reply:
#1829 could probably the venue to discuss this further.  Also ref https://github.com/datalad/datalad/issues/1822#issuecomment-329766900 
meanwhile this one fixes an issue of remove crashing and save/nosave is already present in other command(s), so imho it is consistent in general. indeed `add` should get the same powers, and vise versa.  Also, in principle, all those which save should acquire all the powers of save (e.g. `-S`)

Reply:
Actually, `add` has it already. I was wrong about that.